misc/python/materialize/buildkite_insights/costs/extract_stats.py

```diff
@@ -20,20 +20,29 @@
     "c5.12xlarge": 2.040,
```

Comment:
Where do you get these numbers from? Are they public and stem from the AWS website or are they from somewhere else?

Reply:
https://instances.vantage.sh/aws/ec2/c5.12xlarge